utils/tracking.ts

```diff
@@ -0,0 +1,73 @@
+import { RetrievedService } from "@pagopa/io-functions-commons/dist/src/models/service";
+import { initTelemetryClient } from "./appinsight";
+
+/**
+ * Track when an incoming service document is invalid
+ *
+ * @param telemetryClient
+ * @returns
+ */
+export const trackIgnoredInvalidIncomingDocument = (
```

Comment:
```suggestion
export const trackInvalidIncomingDocument = (
```

Reply:
done in https://github.com/pagopa/io-subscription-migration/pull/13/commits/49dec557ac04572747653e6b9aef86c2a50d9898